appyx-navigation/common/src/commonMain/kotlin/com/bumble/appyx/navigation/node/Node.kt

```diff
@@ -81,17 +81,13 @@ open class Node @VisibleForTesting internal constructor(
 
     private var wasBuilt = false
 
-    val id: String
-        get() = buildContext.identifier
-
-    override val requestCodeClientId: String = id
-
     init {
-        if (BuildConfig.DEBUG) {
-            lifecycle.addObserver(LifecycleLogger)
-        }
-        lifecycle.addObserver(object : DefaultLifecycleObserver {
-            override fun onCreate(owner: LifecycleOwner) {
+        // TODO: expose debug flag
+//        if (BuildConfig.DEBUG) {
+//            lifecycle.addObserver(LifecycleLogger(this))
+//        }
```

Comment:
Let's make a note of this otherwise we will forget

Reply:
implemented BuildFlags as a simple configuration object that we can flesh out later and re-enabled this logging code